Source/WebCore/platform/graphics/skia/GraphicsContextSkia.cpp

```diff
@@ -343,9 +343,8 @@ SkPaint GraphicsContextSkia::createStrokePaint(std::optional<Color> strokeColor)
         // FIXME: Implement stroke patterns.
         UNUSED_PARAM(strokePattern);
         notImplemented();
-    } else if (auto strokeGradient = state.strokeBrush().gradient()) {
-        // FIXME: Implement stroke gradients.
-        UNUSED_PARAM(strokeGradient);
+    } else if (auto strokeGradient = state.strokeBrush().gradient())
```

Comment:
Isn't it missing  a `{`?

Reply:
I messed this up with `git add --patch`, I removed the `{` because I also removed the `notImplemented()` but it was not added. Thanks!